packages/sentry-utils/src/wrap-with-sentry-release.ts

```diff
@@ -0,0 +1,49 @@
+import { execSync } from "node:child_process";
+
+/**
+ * Wraps command with Sentry release tag. Wrapped command is executed using node `execSync` function.
+ *
+ * @param {string} params.cmd - The command to be executed.
+ * @param {string} params.packageVersion - The version of the package.
+ *
+ * @example
+ * wrapWithSentryRelease({ cmd: 'npm run build', packageVersion: '1.0.0' });
+ */
+export const wrapWithSentryRelease = ({
+  cmd,
+  packageVersion,
+}: {
+  cmd: string;
+  packageVersion: string;
+}) => {
+  const release = getReleaseTag(packageVersion);
+
+  console.log("Using release tag:", release);
+
+  execSync(`SENTRY_RELEASE='${release}' ${cmd}`, { stdio: "inherit" });
+};
```

Comment:
I think this is a little too complex - or at least can be simplified?

What you need to do is to compute release tag and set env variable. You don't have to inject build script here

Why don't you have something like this:

```
export const setSentryRelease(packageVersion: string) {
  const release = getReleaseTag(packageVersion);

  console.log("Using release tag:", release);

  process.env.SENTRY_RELEASE = release
}

// in deploy

setSentryRelease(...);
execSync(...)
```

I think its much better to keep spawning scripts inside apps themselves